web-components/pagination/src/makeData.ts

```diff
@@ -0,0 +1,72 @@
+import { faker } from '@faker-js/faker';
+
+export type Resource = {
+  id: string;
+  name: string;
+  rule: string;
+  status: string;
+  other: string;
+  example: string;
+  subRows?: Resource[];
```

Comment:
I'm guessing this is leftover from another example but you don't need `subRows` for this example so we can remove it here.